internal/enums/order_statuses.go

```diff
@@ -0,0 +1,27 @@
+package enums
+
+import "fmt"
+
+type OrderStatus int
```

Comment:
Какой в этом смысл? Я кажется уже писал про это. У нас статусы явно строковые. Зачем их приводить к инту и потом преобразовывать обратно?
```suggestion
type OrderStatus string
```